### PR TITLE
Grey verbal speech now Wingdings

### PR DESCRIPTION
**Pull request by ZomgPonies**
- All grey verbal speech (talk, whisper, radio) now made into Wingdings
- Telepathy uses normal font
- Removed recharge time on telepathy

This should give greys a bit more uniqueness and put the spotlight on their telepathy powers.

### CODE REVIEW

**Comment by Robertha89** [comment]
you need to make a special Comic one for grey, otherwise they will probably speak in comic sans

File: code/modules/mob/living/carbon/human/species/station.dm [hunk 1]
```diff
@@ -360,6 +360,7 @@
 		C.dna.SetSEState(REMOTETALKBLOCK,0,1)
 		C.mutations -= REMOTE_TALK
 		genemutcheck(C,REMOTETALKBLOCK,null,MUTCHK_FORCED)
+	C.mutations.Add(GREY)
```
Comment (tigercat2000) [inline code/modules/mob/living/carbon/human/species/station.dm]: this should probably be `C.mutations |= GREY`, if only for consistancy

File: code/stylesheet.dm [hunk 2]
```diff
@@ -91,6 +91,7 @@ h1.alert, h2.alert		{color: #000000;}
 .rough					{color: #7092BE; font-family: "Trebuchet MS", cursive, sans-serif;}
 .say_quote				{font-family: Georgia, Verdana, sans-serif;}
 .sans					{font-family: "Comic Sans MS", cursive, sans-serif; font-weight: bold;}
+.grey					{font-family:Wingdings", cursive, sans-serif;}
```
Comment (tigercat2000) [inline code/stylesheet.dm]: >`font-family:Wingdings"`
>`:Wingdings"`
>`"`

File: code/__DEFINES/genetics.dm [hunk 3]
```diff
@@ -27,6 +27,7 @@
 
 // Extra powers:
 #define LASER			9 	// harm intent - click anywhere to shoot lasers from eyes
+#define GREY			10	//Wingdings for Greys -- Not activatable or researchable
```
Comment (tigercat2000) [inline code/__DEFINES/genetics.dm]: This being a mutation is.. rather iffy.
I know no systems really exist for species font classes, but one probably should.

**Comment by Plotron** [comment]
So how does this work in practice? Picture, please!

File: code/__DEFINES/genetics.dm (hunk 3 above)
Comment (ZomgPonies) [inline code/__DEFINES/genetics.dm]: The problem is checking a player's species each time he talks is iffier than checking for a mutation. Note that this mutation is the same as LASER eyes e.g. not togglable via Player Panel or researchable through genetics.

File: code/modules/mob/living/carbon/human/species/station.dm (hunk 1 above)
Comment (ZomgPonies) [inline code/modules/mob/living/carbon/human/species/station.dm]: I followed the nomenclature LASER uses throughout the code.

**Comment by ZomgPonies** [comment]
@Robertha89 Nope, Grey takes precedence over Comic. So grey clowns speak in Wingdings and not comic sans.

**Comment by Spacemanspark** [comment]
:+1:

File: code/__DEFINES/genetics.dm (hunk 3 above)
Comment (tigercat2000) [inline code/__DEFINES/genetics.dm]: yeah, there's no really clean solution for this, the mutation system is just kinda weird in general

**Comment by Mijhead** [comment]
:+1: :x:  :100:

**Comment by Finalsong** [comment]
:+1: :x: :100:

**Comment by Regen1** [comment]
But.. Other grey's should be able to understand other grey's.
Does that still apply?

**Comment by mrsdobbins** [comment]
I will now actually use my grey. :+1: +1 :x: :100: .

**Comment by ZomgPonies** [comment]
No that would require a huge overhaul of languages. I looked it over and scambling while using a different font would be....ridiculously hard.

**Comment by tigercat2000** [comment]
well you could always just copypaste the wingdings into something and switch it back to normal text

**Comment by KasparoVy** [comment]
I was thinking it'd be better applied to plasmen-- but I'd not even thought once about greys! Excellent idea

**Comment by FalseIncarnate** [comment]
Fucking Wingdings.

:+1:

**Comment by NTSAM** [comment]
Beware the man who speak in hands.

**Comment by taukausanake** [comment]
Man, this is what I wanted for Plasmamen. Oh well, at least people can say that greys are becoming more special

**Comment by fludd12** [comment]
Can we remove the telepathy cooldown, then? It gets annoying already for someone who mains a mute grey - the same being applied to every grey will make communication a little pissy.

**Comment by ZomgPonies** [comment]
Yep I'll do that when I wake up tomorrow. Remove cooldown for greys but not for non-greys with the genetic power

**Comment by Iamgoofball** [comment]
Why not make this available as a genetic defect?

**Comment by ZomgPonies** [comment]
Because genetics already have enough, this is a unique feature meant for a specific species. It's only genetic due to the fact it's less messy in code.

**Comment by Fox-McCloud** [comment]
Don't think it's a good idea to make telepathy 0 CD when it's a global ability.

I'd say that if you're going to do that, then it should only work with people in vision range.

**Comment by Robertha89** [comment]
or make it consume nutriment

**Comment by TheDZD** [comment]
@Robertha89 That's never a very good downside, it doesn't prevent you from going global range unlimited antag yeller while you're disabled, and food is fairly easy to get a hold of. Not to mention it's their only decent way to communicate with their speech being made into wingdings.

**Comment by fludd12** [comment]
Telepathy isn't truly global anymore, since it's now limited by Z-Level, but I can see your point, Fox.

**Pull request merged**